src/css/index.scss

```diff
@@ -0,0 +1,105 @@
+section.contact {
+  padding-bottom: 100px;
+  padding-top: 10px;
+}
+section.contact h2 {
+  font-size: 38px;
+  font-size-adjust: inherit;
+  font-family: avenir, sans-serif;
```

Comment:
`avenir` is not web safe.

I will give you credit to getting the required font, I don't mind you put the font in the font family but you do need to improve your fall back font-family. 

Please refer to our modal answer for this:
```
// font family: https://ant.design/docs/spec/font#Font-Family
$font-family-content: -apple-system, BlinkMacSystemFont, 'Segoe UI',
  'PingFang SC', 'Hiragino Sans GB', 'Microsoft YaHei', 'Helvetica Neue',
  Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji',
  'Segoe UI Symbol';
$font-family-title: Avenir, $font-family-content;
```

Reply:
Also you should define a default font family on body, only override it on some elements that requires.